src/main/java/com/teragrep/pth_06/config/ConditionConfig.java

```diff
@@ -86,14 +86,29 @@ public boolean withoutFilter() {
 
     @Override
     public boolean equals(Object object) {
-        if (this == object)
+        if (this == object) {
             return true;
-        if (object == null)
+        }
+        if (object == null) {
             return false;
-        if (object.getClass() != this.getClass())
+        }
+        if (object.getClass() != this.getClass()) {
             return false;
+        }
         final ConditionConfig cast = (ConditionConfig) object;
         return this.bloomEnabled == cast.bloomEnabled && this.streamQuery == cast.streamQuery
                 && this.withoutFilters == cast.withoutFilters && this.ctx == cast.ctx;
     }
+
+    @Override
+    public int hashCode() {
```

Comment:
I think `Objects.hash()` could be used instead. Idea should generate these (equals and hashCode) automatically

Reply:
Solved in commit 4d4e0023426356b05c592147f65933f69ba5dfa3